tensorflow/core/kernels/BUILD

```diff
@@ -103,7 +103,6 @@ tf_kernel_library(
         "strided_slice_op.h",
         "strided_slice_op_impl.h",
         "strided_slice_op_gpu.cu.cc",
-        "slice_op_gpu.cu.cc",
```

Comment:
Yes, I think slice_op_gpu.cu.cc should not be here.
It's already the source of slice_op during gpu build, it'll be compiled twice in different targets.
And then I got this linking error:

```
slice_op_gpu.cu.o : error LNK2005: "unsigned __int64 __cdecl Eigen::internal::_GLOBAL__N__34_slice_op_gpu_cu_compute_52_cpp1_ii_b2f7d5b8::get_random_seed(void)" (?get_random_seed@_GLOBAL__N__34_slice_op_gpu_cu_compute_52_cpp1_ii_b2f7d5b8@internal@Eigen@@YA_KXZ) already defined in slice_op_gpu.cu.o
```